Neos.Neos/Classes/PendingChangesProjection/Change.php

```diff
@@ -30,13 +30,16 @@
  */
 final class Change
 {
+    public const AGGREGATE_DIMENSIONSPACEPOINT_HASH_PLACEHOLDER = 'AGGREGATE';
+
     /**
      * @param NodeAggregateId|null $removalAttachmentPoint {@see RemoveNodeAggregate::$removalAttachmentPoint} for docs
      */
     public function __construct(
         public ContentStreamId $contentStreamId,
         public NodeAggregateId $nodeAggregateId,
-        public OriginDimensionSpacePoint $originDimensionSpacePoint,
+        // null for aggregate scoped changes (e.g. NodeAggregateNameWasChanged, NodeAggregateTypeWasChanged)
+        public ?OriginDimensionSpacePoint $originDimensionSpacePoint,
```

Comment:
todo the neos ui must learn to handle this case: https://github.com/neos/neos-ui/blob/13520defa0b00eee9cef6073fde11ccbfa3346d4/Classes/ContentRepository/Service/WorkspaceService.php#L83

i assume we need to mark all occupied nodes of the noteaggregate as changes then? And if `removalAttachmentPoint` is set is probably gurantted that `originDimensionSpacePoint` is set